linux_pipeline/Jenkinsfile_upstream_kernel

```diff
@@ -506,6 +522,22 @@ pipeline {
             always {
               junit "Report\\*-junit.xml"
               archiveArtifacts "TestResults\\**\\*"
+              script {
+                 withCredentials(bindings: [
+                 file(credentialsId: 'CBS_Azure_Secrets_File',
+                       variable: 'Azure_Secrets_File')
+                ]) {
+                        def failures=powershell returnStdout: true, script: '''$content = [xml](Get-Content ".\\Report\\*-junit.xml")
+                            $failures_count=[int]($content.testsuites.testsuite.failures)
+                            return $failures_count
+                            '''
+                        failures=failures.replaceAll("\r", "").replaceAll("\n", "")
+                        echo "${failures}"
+                        sql_string="update Linux_Kernel_Quality set FuncTestsFailedOnAzure=${failures} where PipelineBuildNumber=${env.BUILD_NUMBER}"
+                        echo "${sql_string}"
+                        RunPowershellCommand("cd Utilities;.\\UploadDataToDataBase.ps1 -SQLQuery '${sql_string}' -AzureSecretsFile '${Azure_Secrets_File}'")
```

Comment:
this is duplicate code

Reply:
one is for FuncTestsFailedOnAzure, the other is for FuncTestsFailedOnLocal